DB design

lsaqrxq opened the pull request:
# WHAT
DB設計用のマークダウン。

# WHY
開発に必須の手順であるため。

Commit:
Merge branch 'master' into db-design

Commit:
Edit DB design 3

miyagit reviewed (commented):
has_manyでdependent: :destroyが付けられそうなところは付けましょう！

Commit:
Update README.md

# WHAT
・dependent destroyを追記
# WHY
・どのテーブルのレコードが削除されたら関連のあるテーブルレコードを削除するかを明確にするため

wheesnoza commented:
修正しました。以下にER図があります。
＜https://i.gyazo.com/ac1af2fffeaadb9b053b16662b7eb5e4.png＞

miyagit reviewed (commented):
LGTM です。
あとは変更点が出てきたら随時DB設計を変更していきましょう。
